docs/angular/api-node/generators/migrate-to-webpack-5.md

```diff
@@ -0,0 +1,37 @@
+# migrate-to-webpack-5
+
+Add webpack 5 compatible dependencies to the workspace
+
+## Usage
+
+```bash
+nx generate migrate-to-webpack-5 ...
+```
+
+```bash
+nx g webpack5 ... # same
+```
+
+By default, Nx will search for `migrate-to-webpack-5` in the default collection provisioned in `angular.json`.
```

Comment:
should this say `angular.json` or `workspace.json`, considering that this guide is for node? 

same question for `api-web`